etc/DFLMessages_en_GB.dic

```diff
@@ -37,16 +37,26 @@ RefUnsupportedTag             =     Settings ref %1% uses a unrecognized tag %2%
 
 //------------------ Main ---------------------------
 NetworkSlackNodeNotFound      =     Network slack node requested but not found in network input file %1%
-InputsInfo                    =     Processing network input file %1% and config file %2%
+InputsSteadyStateInfo         =     Processing network input file %1% and config file %2% for steady-state calculation
+InputsSecurityAnalysisInfo    =     Processing network input file %1%, contingencies %2% and config file %3% for security analysis
 ExportInfo                    =     Exporting outputs files for %1%
 SlackNode                     =     Slack node of id %1% found with origin %2% (0=file, 1=algorithm)
 ConnexityError                =     Slack node of id %1% not present in main connex component
 ConnexityErrorReCompute       =     Slack node of id %1% not present in main connex component: compute slack node only in main connex component
 SimulateInfo                  =     Initializing %1% simulation
 NetworkFileNotFound           =     Network file %1% does not exist
 NetworkHasNoRegulatingGenerator         =     Network file %1% has no generator regulating voltage in the main connex component
+ContingenciesFileNotFound     =     Contingencies file %1% was given but does not exist
+SimulationKindUnknown         =     Simulation kind (steady-state calculation or security analysis) could not be determined %1%
 ContextProcessError           =     Processing of the network %1% failed
 SimulationEnded               =     Simulation %1% ended successfully (wall-time: %2%s)
 DFLEnded                      =     DynaFlowLauncher %1% ended successfully (wall-time: %2%s)
 InitEnd                       =     End of initialization (wall-time: %1%s)
 FilesEnd                      =     End of files generation (wall-time: %1%s)
+
+//-------------- Contingencies -----------------------
+ContingenciesReadingFrom      =     Reading contingencies from %1% ...
+ContingenciesReadError        =     Error while reading contingencies file %1%: %2%
+ContingencyInvalidBadElemType =     Contingency %1% is invalid: element %2% has an unknown type "%3%"
+ContingencyInvalidForSimulation =     Contingency %1% is invalid for simulation: element %2% invalid "%3%"
```

Comment:
Parameter %3% is the "reason" that explains why it is invalid (the enum `ValidationStatus` converted to a string). It could be the element not in the main connected component or not defined in the contingency with a valid type. I will clarify the message, but I think it is easier if we just dump the string representation of the enum.

Reply:
OK that's what I thought. For me , only somethink like "reason: " or ":" after invalid is needed to clarify it enought